sdk/tables/azure-data-tables/CMakeLists.txt

```diff
@@ -70,10 +70,11 @@ set(
     src/private/policies/tenant_bearer_token_policy.hpp
     src/private/policies/timeout_policy.hpp
     src/private/serializers.hpp
+    src/private/table_constants.hpp
```

Comment:
I'd think `tables_constants` (plural) would be a better name, because Tables is a name of an Azure service, while Table is one of the abstractions of that service.

Reply:
Sure.